File: src/nomanssave/fG.java
package nomanssave;

import java.util.Comparator;

class fG implements Comparator {
   // $FF: synthetic field
   final fE mf;

   fG(fE var1) {
      this.mf = var1;
   }

   public int a(fs var1, fs var2) {
      long var3 = var2.lastModified() - var1.lastModified();
      if (var3 < -2147483648L) {
         return Integer.MIN_VALUE;
      } else {
         return var3 > 2147483647L ? Integer.MAX_VALUE : (int)var3;
      }
   }

   // $FF: synthetic method
   public int compare(Object var1, Object var2) {
      return this.a((fs)var1, (fs)var2);
   }
}

File: NMSSaveEditor/nomanssave/mixed/fG.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace NMSSaveEditor
{

public class fG : IComparer<object> {
   fE mf;

   public fG(fE var1) {
      this.mf = var1;
   }

   public int a(fs var1, fs var2) {
      long var3 = var2.LastWriteTimeUtc.Ticks - var1.LastWriteTimeUtc.Ticks;
      if (var3 < -2147483648L) {
         return int.MinValue;
      } else {
         return var3 > 2147483647L ? int.MaxValue : (int)var3;
      }
   }
   public int Compare(object var1, object var2) {
      return this.a((fs)var1, (fs)var2);
   }
}

}
